test/sanity/list.wtest

```diff
@@ -109,8 +109,8 @@ describe "List tests"{
 	}	
 
 	test "without duplicates" {
-		assert.equals([1, 3, 1, 5, 1, 3, 2, 5].withoutDuplicates(), [1, 3, 5, 2])
-		assert.equals([1, 3, 5, 2].withoutDuplicates(), [1, 3, 5, 2])
+		assert.equals([1, 3, 5, 2], [1, 3, 1, 5, 1, 3, 2, 5].withoutDuplicates())
```

Comment:
Ok, I already fixed that on my branch just to make sure we don't loose this 2 lines.